Makefile targets for individual MIDAS-level tests broken

Building a single test doesn't work as expected - there seems to be an issue with the Makefile rule. I entered the following:
```
make veryclean
make DESIGN=FireSimNoNIC
make DESIGN=FireSimNoNIC verilator-debug
make $(pwd)/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.vpd
```
resulting in the following error
`make: *** No rule to make target '/home/centos/firesim/sim/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.vpd'.  Stop.`
However, `make -j run-asm-tests` works as expected.

--- a/docs/Advanced-Usage/Debugging/RTL-Simulation.rst
+++ b/docs/Advanced-Usage/Debugging/RTL-Simulation.rst
@@ -109,7 +109,7 @@ Run rv64ui-p-simple (a single assembly test) on a verilated simulator.
 ::
 
     make DESIGN=FireSimNoNIC
-    make $(pwd)/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.out
+    make DESIGN=FireSimNoNIC $(pwd)/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.out
 
 Run rv64ui-p-simple (a single assembly test) on a VCS simulator with waveform dumping.
 
@@ -117,7 +117,7 @@ Run rv64ui-p-simple (a single assembly test) on a VCS simulator with waveform du
 
 
     make DESIGN=FireSimNoNIC vcs-debug
-    make EMUL=vcs $(pwd)/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.vpd
+    make DESIGN=FireSimNoNIC EMUL=vcs $(pwd)/output/f1/FireSimNoNIC-FireSimRocketChipConfig-FireSimConfig/rv64ui-p-simple.vpd
 
 
 FPGA-Level Simulation

